lib/twiglet/logger.rb

```diff
@@ -37,7 +38,7 @@ def error(message = nil, error = nil, &block)
           }
         }
         add_stack_trace(error_fields, error)
-        message = message.is_a?(Hash) ? message.merge(error_fields) : error_fields.merge(message: message)
+        message = Message.format(message).merge(error_fields)
```

Comment:
We could add a `merge` method to the Message class, but I didn't think it was necessary.

Reply:
Agree

Reply:
See below comment about inheriting from Hash.